src/dune_rules/inline_tests.ml

```diff
@@ -264,7 +264,18 @@ include Sub_system.Register_end_point (struct
             Resolve.Memo.List.concat_map backends ~f:(fun (backend : Backend.t) ->
               backend.runner_libraries)
           in
-          let* lib = Lib.DB.resolve lib_db (loc, Library.best_name lib) in
+          let* arguments =
+            Resolve.Memo.lift_memo
+            @@ Memo.List.map info.arguments ~f:(fun (loc, dep) ->
+              let open Memo.O in
+              let+ dep = Lib.DB.resolve lib_db (loc, dep) in
```

Comment:
Is there a reason to not also use `Library.best_name` on the `dep` here?

Reply:
Here the `dep` is already a `Lib_name.t` (~ a string, as written by the user in the stanza) so it's already the "best name" to look it up in the database :) (below we have `lib:Library.t` for which there could be multiple names to use when searching the db for its resolved `Lib.t`, so we need to pick one)